build/repo.props

```diff
@@ -12,7 +12,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <DotNetCoreRuntime Include="$(MicrosoftNETCoreApp20PackageVersion)" />
-    <DotNetCoreRuntime Include="$(MicrosoftNETCoreApp21PackageVersion)" />
+    <DotNetCoreRuntime Include="$(MicrosoftNETCoreApp22PackageVersion)" />
```

Comment:
@natemcmaster super unrelated - but is it possible to move installing NETFx on xplat as a requirement here? If we remove `net461` from appearing in the sample on xplat, we could make it so that it's only installed on repos that cross-compile sources.

Reply:
Would be cool, but not something we can automate AFAIK. Netfx *runtime* installations require admin privileges.

What would be possible, though, is using netfx *reference assemblies* from a package instead of requiring them to be installed on the machine. RepoToolset (dotnet/arcade) already has support for this. I'm hoping to adopt it soon-ish.

Reply:
I don't see a sudo here - https://github.com/aspnet/BuildTools/blob/dev/files/KoreBuild/KoreBuild.sh#L93-L99?

Reply:
This script installs netfx reference assemblies for Linux only. This is what I meant by providing netfx reference assemblies from a nuget package. See https://github.com/aspnet/BuildTools/issues/399 for some more context on this. We decided not to do anything at the moment since there are tentative plans to build this into Microsoft.NET.Sdk. See https://github.com/dotnet/sdk/issues/335 and https://github.com/dotnet/designs/pull/33